README.md

```diff
@@ -90,44 +89,7 @@ func main() {
 		panic(err)
 	}
 
-	fmt.Println(scout)
-}
-```
-
-### Composing Multiple Queries
```

Comment:
What's your motivation for removing this?

Reply:
a user cannot just copy and paste it and run it

Reply:
Then I think we should update the example rather than remove it. Showing query composition is important imo...